ambry-replication/src/main/java/com/github/ambry/replication/ReplicaThread.java

```diff
@@ -2050,6 +2071,12 @@ public void handleGetResponse(ResponseInfo responseInfo) {
           // Deserialize the request from the given ResponseInfo
           DataInputStream dis = new NettyByteBufDataInputStream(responseInfo.content());
           GetResponse response = GetResponse.readFrom(dis, clusterMap);
+          if (response.getError() != ServerErrorCode.No_Error) {
+            // Getting error for entire response, something unexpected happened.
+            logger.error("Remote node: {} Thread name: {} RemoteReplicaGroup {} ServerError for GetResponse {}",
+                remoteDataNode, threadName, id, response.getError());
```

Comment:
nit: Similar to error message in `getReplicaMetadataResponse`, would it be helpful if we print the list of remote replicas too.

Reply:
the replicas will be printed out later